### PR TITLE
Feat/filter: Changed some files

### PR DESCRIPTION
**Pull request by veerleprins**
Hey guys, I first made some adjustments in route.js (for example, changed some comments). Just changed the POST method in filter.ejs, the p tags, etc so that everything works. In the home.ejs I have added some text when no users are found after the filters / everyone from the database is liked. In CSS the button is styled on the filter.ejs page so that the user can remove the filters.

### CODE REVIEW

File: routes/route.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ mongo.MongoClient.connect(url, {
 router.get('/', signIn); // Rowan, eerste pagina (index)
 router.get('/registration', registration); // Rowan klaar
 router.post('/registration', createAccount); // Rowan klaar
-router.post('/home', logIn); // Rowan
+// router.post('/profile', logIn); // Rowan
```
Comment (joordy) [inline routes/route.js:31]: Good! Don't forget the correct routing @rowanhorn1412!

File: view/404.ejs [hunk 2]
```diff
@@ -1 +1,20 @@
...
+    <%- include ("components/header-two.ejs") %> 
+        <section>
+            <h2>Ooooops</h2>
+            <img src="images/icons/error.svg" alt="Warning sign">
+            <p>Sorry, the page you are trying to reach does not exist...Go back to the home page.</p>
+            <a href="/home">Go to homepage</a>
+        </section>
+</body>
+
+</html>
```
Comment (joordy) [inline view/404.ejs:20]: Nice and subtile page! Looking good and it matches the style of the project. I like!

File: view/filter.ejs [hunk 3]
```diff
@@ -3,67 +3,68 @@
 <html lang="en">
 
 <head>
-    <%- include ("components/head.ejs") %>
-        <title>Dat.io – Filters</title>
+    <%- include ("components/head.ejs") %> 
+    <title>Filters - Dat.io</title>
```
Comment (joordy) [inline view/filter.ejs:7]: Logic. Do you know if this is used on every page?

Comment (rowanhorn1412) [reply view/filter.ejs:7]: i know most of them do, if everybody checks their own .ejs files its okay

File: routes/route.js [hunk 4]
```diff
@@ -90,11 +90,10 @@ async function logIn(req, res, next) {
   // Rowan
   try {
     //Veerle: Rowan, hierin moet een session beginnen met de 
-    //globale: loggedInUser. < dit is de ingelogde gebruiker.
+    //globale: idLoggedIn. < dit is de ingelogde gebruiker.
     //Voor nu zet ik er even static code in zodat mijn code alvast kan werken: 
-    req.session.gender = 'everyone';
-    req.session.movie = '';
-    loggedInUser = 5;
```
Comment (rowanhorn1412) [inline routes/route.js:97]: are the comment files the static files, or are you still need to delete them?

**Pull request merged**